admin/app/components/solidus_admin/ui/pagination/component.rb

```diff
@@ -1,4 +1,42 @@
 # frozen_string_literal: true
 
 class SolidusAdmin::UI::Pagination::Component < SolidusAdmin::BaseComponent
+  def initialize(page:, path: nil)
```

Comment:
It took me a couple of minutes to understand that `path` is a `Proc`. What do you think if we add yard tags for the `#initialize` parameters to make it super clear?